### PR TITLE
[Notifier] Add section for editing telegram messages

### LINKED ISSUE #17391: [Notifier] Add support for editing Telegram messages
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#47956
| PR author(s) | @chr-hertel
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by chr-hertel**
Fix #17391

### CODE REVIEW

**Comment by chr-hertel** [comment]
I respectfully tend to disagree ... what do I get wrong? :D

Bildschirmfoto von 2022-10-23 13-49-23

File: notifier/chatters.rst [hunk 1]
```diff
@@ -306,6 +306,27 @@ to add `message options`_::
 
     $chatter->send($chatMessage);
 
+Updating Telegram messages
```
Comment (wouterj) [inline notifier/chatters.rst]:
```suggestion
Updating Telegram Messages
```

(our title case is a bit confusing, but every "non-closed word" must be capitalized)

File: notifier/chatters.rst [hunk 2]
```diff
@@ -306,6 +306,27 @@ to add `message options`_::
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When working with interactive callback buttons, you can use the
+`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` to reference
```
Comment (wouterj) [inline notifier/chatters.rst]:
```suggestion
:class:`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` to reference
```

File: notifier/chatters.rst [hunk 3]
```diff
@@ -306,6 +306,27 @@ to add `message options`_::
...
+
+    $chatMessage = new ChatMessage('Are you really sure?');
+    $telegramOptions = (new TelegramOptions())
+        ->chatId($chatId)
+        ->edit($messageId) // extracted from callback payload or SentMessage
+        ->replyMarkup((new InlineKeyboardMarkup())
+            ->inlineKeyboard([
+                (new InlineKeyboardButton('Absolutely'))->callbackData('yes'),
+            ])
+        );
```
Comment (wouterj) [inline notifier/chatters.rst:329]:
```suggestion
        );

.. versionadded:: 6.2

    The ``TelegramOptions::edit()`` method was introduced in Symfony 6.2.

```

**Review by wouterj — COMMENTED** [review]
Thanks!

> I respectfully tend to disagree ... what do I get wrong? :D

Nothing :) Code blocks are checked within a real project (https://github.com/symfony-tools/symfony-application) and I think we are missing a dependency there.

**Review by OskarStark — APPROVED** [review]

**Comment by wouterj** [comment]
Thank you @chr-hertel!

**Pull request merged**